### PR TITLE
Blog: Install Files with WinGet

### PR DESCRIPTION
**Pull request by Lamparter**
## Description

Added blog post about installing Files via WinGet.

## Motivation and Context

Requested in Files Discord server

## Screenshots:

image

### CODE REVIEW

File: src/routes/blog/posts/now-on-winget/+page.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+---
+title: Files is now available on WinGet again
+description: Files is back on WinGet! I'm pleased to announce that Files can finally be installed from WinGet again.
+thumbnail: /blog-resources/now-on-winget/Hero.jpg
+date: 11/17/2024
+author: Lamparter
+---
+
+I'm are excited to announce that the app Files is now available again on WinGet! After initially being removed, Files has made a triumphant return, offering users an enhanced and seamless file management experience on Windows.
```
Comment (yaira2) [inline src/routes/blog/posts/now-on-winget/+page.md:9]:
```suggestion
I'm excited to announce that Files is now available again on WinGet! After a couple years, we're pleased to bring back this popular installation method for Files.
```

File: src/routes/blog/posts/now-on-winget/+page.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+### Files and WinGet
+
+The return of Files to WinGet signifies the continued commitment to providing users with the best experience in offering convenient, friendly installation methods compatible with their device.
```
Comment (yaira2) [inline src/routes/blog/posts/now-on-winget/+page.md:13]:
```suggestion
The return of Files to WinGet signifies our continued commitment to providing users with the best experience, allowing them to install and manage their apps the way they desire.
```

File: src/routes/blog/posts/now-on-winget/+page.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+WinGet allows users to easily install, update, and manage software packages on their Windows devices. By integrating Files into WinGet, users can now enjoy a hassle-free installation process and keep their file manager up to date with the latest features and improvements.
+
```
Comment (yaira2) [inline src/routes/blog/posts/now-on-winget/+page.md:16]:
```suggestion
```

**Comment by Lamparter** [comment]
image

<details>
<summary>Full post</summary>
<p>

image

</p>
</details>

File: src/routes/blog/posts/now-on-winget/+page.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
+---
+title: Files is now available on WinGet again
```
Comment (yaira2) [inline src/routes/blog/posts/now-on-winget/+page.md:2]:
```suggestion
title: Files is back on WinGet
```

Comment (yaira2) [inline src/routes/blog/posts/now-on-winget/+page.md:2]:
```suggestion
title: Files is now available on WinGet
```

File: src/routes/blog/posts/now-on-winget/+page.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+author: Lamparter
+---
+
+I'm excited to announce that Files is now available again on WinGet! After a couple years, we're pleased to bring back this popular installation method for Files.
+
+### Files and WinGet
+
+The return of Files to WinGet signifies our continued commitment to providing users with the best experience, allowing them to install and manage their apps the way they desire.
+
+### How to Install Files via WinGet
```
Comment (yaira2) [inline src/routes/blog/posts/now-on-winget/+page.md:15]:
```suggestion
### How to install Files via WinGet
```

**Review by yaira2 — APPROVED** [review]
LGTM

**Pull request merged**